### PR TITLE
change card structure to allow for accessability tweaks

### PR DESCRIPTION
**Pull request by Skadefryd**
To make te accessability stuff and also styling options viable I saw 2 solves
1. move  the content inside the component itself, looked to rigid and less likely we could reuse it for other things. 
2. (Chosen solve )move the content bit outside of the card, means more styling in the actual use of the component but I think that is fine since we are the only consumer of the thing. Allows alot of flexibility. 

This thing is styled with unoclasses for now.

### CODE REVIEW

File: docs/components/alert/data.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "title": "Alert",
+  "mainurl": "/warp-portal-poc/components/alert/",
```
Comment (Skadefryd) [inline docs/components/alert/data.json:3]: I added this, is it already available elsewhere?

Comment (BalbinaK) [reply docs/components/alert/data.json:3]: Yes, it's dynamically set in Component Overview's [index.md's script tag](https://github.com/warp-ds/warp-portal-poc/blob/main/docs/components/index.md?plain=1#L8).

Comment (BalbinaK) [reply docs/components/alert/data.json:3]: You should be able to read the url path under `component.href` in `index.md`

Comment (Skadefryd) [reply docs/components/alert/data.json:3]: Isnt that just the first bit of it though?  Is the specifik path to  /components/alert/ available when it loops through the cards?

File: docs/.vitepress/Card.vue [hunk 2]
```diff
@@ -11,29 +11,17 @@
...
 
-<style lang="scss">
-.card-wrapper {
-  .img-bg {
-    background-color: #F9F9FA;
-  }
+<style scoped lang="scss">
+
+.docs-card:focus-within{
+  //border: 1px solid red;
```
Comment (BalbinaK) [inline docs/.vitepress/Card.vue:23]: Do we want to keep it if it's commented out? I see there's no navigation feedback now, but if you restore it, the card changes width on focus:
card-focus

**Review by BalbinaK — APPROVED** [review]

File: docs/.vitepress/Card.vue (hunk 2 above)
Comment (Skadefryd) [reply docs/.vitepress/Card.vue:23]: The staes are just guesswork for now, will go over it when that stuff is ready. There is actually navigation feedback as a shadow thing, hard to see in dark mode, but dark mode is not on the supportplan right now anyways :)

**Pull request merged**